crates/bevy_asset/src/lib.rs

```diff
@@ -75,6 +76,8 @@ pub struct AssetPlugin {
     pub watch_for_changes_override: Option<bool>,
     /// The [`AssetMode`] to use for this server.
     pub mode: AssetMode,
+    /// How meta files should be accessed. This will be ignored for processed assets, as this mode require meta files.
+    pub meta_mode: AssetMetaMode,
```

Comment:
Adding a field to a struct that isn't marked `#[non_exhaustive]` is a breaking change. The signature change of `AssetServer::new` is also breaking. Perhaps this should be an overridable `Resource` instead?

Reply:
Hmmm yeah fair points. A bit regrettable that we need to break the people adopting this feature (which doesn't yet exist) in 0.13, but thats _probably_ better than breaking someone in a patch release.